nixos/modules/programs/minipro.nix

```
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.minipro;
+in
```

```suggestion
let
  cfg = config.programs.minipro;
  
  inherit (lib)
    mkEnableOption
    mkOption
    mdDoc
    literalExpression
    types
    mkIf;
in
```

I don't mind changing it to explicitly `inherit` them, if that would be preferable :)

(Can also just put `lib` in front of them, not like that isn't an option either)

---

I just put `lib.` in front of everything myself, but it's probably fine as-is (tbh the inherits clutter more than they help usually)

I don't believe we have any hard rules on it yet, so I'd say probably leave it unless someone else says otherwise